community templates/Sensitive Data Discovery with Delphix/manifest.json

```diff
@@ -26,13 +26,24 @@
             }
         }
     },
-    "author": "Delphix",
+    "author": "kishlay.singh@azureopsdelphix.onmicrosoft.com",
```

Comment:
Can you please change the author to Delphix

Reply:
corrected the value